src/test/java/core/dto/LoginUserDto.java

```diff
@@ -0,0 +1,13 @@
+package core.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
@Setter annotation could be deleted if it is not used

Reply:
yes, you right, removed